package/functions/funcs/latestVersion.js

```diff
@@ -1,15 +1,12 @@
+const axios = require('axios')
+
 module.exports = async d => {
-  /*axios.get('https://api.leref.ga/package/version')
+  axios.get('https://api.leref.ga/package/version')
```

Comment:
It's better to use the [npm Registry](https://registry.npmjs.org) instead of Third party API.